src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -1,6 +1,6 @@
 
 export interface ApproveDataOptions {
-    orderID: string;
+    orderID: string | undefined;
```

Comment:
we tend to use `orderID?: string` rather than `string | undefined`.

Reply:
fixed